### PR TITLE
feat(showcase): embed section

### LINKED ISSUE #88: Create Vimeo Component
Section for showcase where we can add vimeo links for video.

### PR DESCRIPTION
**Pull request by danilowoz**
Closes #88

### CODE REVIEW

File: src/components/Projects/Section/types.ts [hunk 1]
```diff
@@ -111,6 +111,11 @@ export interface IStickyImage extends IStickyBase {
 
 export type ISticky = IStickyVideo | IStickyImage
 
+// Embed
+export interface IEmbed {
```
Comment (pbrandone) [inline src/components/Projects/Section/types.ts]: missing 'embed' in sectionTypes

**Comment by pbrandone** [comment]
Deploy preview ready! https://pr-89.significa.io/

File: src/components/Projects/Section/sections/Embed/styled.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import styled from '@theme'
+
+export const Wrapper = styled.div`
+  position: relative;
+  padding-bottom: 56.25%; /* 16:9 */
+  padding-top: 25px;
```
Comment (pbrandone) [inline src/components/Projects/Section/sections/Embed/styled.ts]: why this padding-top? I believe we don't have padding in any section and the sectionWrapper takes care of all the top and bottom spacings

File: src/components/Projects/Section/sections/Embed/styled.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import styled from '@theme'
+
+export const Wrapper = styled.div`
+  position: relative;
+  padding-bottom: 56.25%; /* 16:9 */
```
Comment (pbrandone) [inline src/components/Projects/Section/sections/Embed/styled.ts]: Can we safely assume that it's always 16:9?

cc @pedroet

File: src/templates/project.tsx [hunk 4]
```diff
@@ -322,6 +322,9 @@ export const query = graphql`
               # Highlight
 
               #text
+
+              # Embed
+              code
```
Comment (pbrandone) [inline src/templates/project.tsx]: Gatsby's GraphQL creates schemas automatically based on the queries.
Unfortunately, if we query `code` here, we need to make sure that at least one project .yml has this key.
Add it to the sample project so we don't have any problems with GraphQL

File: src/components/Projects/Section/sections/Embed/styled.ts (hunk 2 above)
Comment (danilowoz) [reply src/components/Projects/Section/sections/Embed/styled.ts]: These styles are about this: https://css-tricks.com/NetMag/FluidWidthVideo/Article-FluidWidthVideo.php

**Comment by pbrandone** [comment]
Deploy preview ready! https://pr-89.significa.io/

**Comment by pbrandone** [comment]
Deploy preview ready! https://pr-89.significa.io/

**Pull request merged**